resource-service/src/test/java/uk/gov/gchq/palisade/service/resource/service/SimpleResourceServiceTest.java

```diff
@@ -43,12 +43,11 @@
 
 @RunWith(JUnit4.class)
 public class SimpleResourceServiceTest {
-    private final ClientConfiguration clientConfiguration = Mockito.mock(ClientConfiguration.class);
-    private final SimpleResourceService service = new SimpleResourceService(clientConfiguration);
+    private final DiscoveryClient discoveryClient = Mockito.mock(DiscoveryClient.class);
+    private final SimpleResourceService service = new SimpleResourceService("data-service");
 
     @Before
     public void setUp() throws URISyntaxException {
-        Mockito.when(clientConfiguration.getClientUri(Mockito.anyString())).thenReturn(Optional.of(new URI("http://data-service-uri")));
     }

```

Comment:
remove method since it is otherwise empty

Reply:
removed